app/styles/superfish.css

```diff
@@ -1,152 +1,113 @@
 /*** ESSENTIAL STYLES ***/
-.sf-menu,
-.sf-menu * {
-    margin: 0;
-    padding: 0;
-    list-style: none;
+.sf-menu, .sf-menu * {
+	margin: 0;
+	padding: 0;
+	list-style: none;
 }
```

Comment:
@jrwdunham i keept this diff here for you to see the result of using superfish's css from bower vs the one you had in teh app. much of it looks like formatting changes or small upgrades but you should check to see if the app behaves abnromally and if yes, we should revert this commit that uses superfish's css from bower